### ISSUE TITLE
Tag 7.0.0-beta.0 as "beta" on npm

### ISSUE THREAD

**Issue by Floby** [feross/eslint-config-standard#71]
eslint-config-standard@7.0.0-beta.0 isn't tagged as "beta" on npm and is therefore automatically marked as "latest".
This causes automated checking of dependencies to give false positives.

**Comment by feross** [comment]
I did this intentionally to get more eyeballs on the new version, but I didn't realize it would false-alarm automated tools. Added a dist-tag.

**Comment by feross** [comment]
Here's what I get when checking directly with npm:

```bash
$ npm show eslint-config-standard dist-tags
{ latest: '6.2.1', beta: '7.0.0-beta.0' }
```

Sounds like either a caching issue, a bug in `npm-check-updates`, or maybe you're accidentally passing one of the following flags:

```
-n, --newest             find the newest published versions available instead 
                         of the latest stable versions
-t, --greatest           find the highest versions available instead of the
                         latest stable versions 
```
